README.md

```diff
@@ -78,6 +79,42 @@ foreach ($staticSitemapUrls as $sitemapUrl) {
 $index->write();
 ```
 
+Multi-language sitemap
+----------------------
+
+```php
+use samdark\sitemap\Sitemap;
+
+// create sitemap
+// be sure to pass `true` as second parameter to specify xhtml namespace
```

Comment:
Should it be done automatically in case multi-language item is added?

Reply:
It will be nice, but can't be done automatically, because we can not skip case when first item is not multi-lingual and the second and others are.